unitOneAssessmentPractice.js

```diff
@@ -3,53 +3,123 @@ let assert = require('assert')
 // Question One:
 
 // Write a function called double that doubles a number
+const double = num => num*2
+
 
 // Uncomment out the next line to test your solution
-// runQ1Tests()
+runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
+const containsSeven = num =>{
+  let number = String(num);
+  if(number.includes("7")){
+    return true;
+  } else {
+    return false;
+  }
+}
+
 
 // Uncomment out the next line to test your solution
-// runQ2Tests()
+runQ2Tests()
 
 // Question Three:
 
 // Write a function called capitalizeTheAs the capitalizes all of the lowercase "a"s in a string leaving all other characters the same
+const capitalizeTheAs = str =>{
+    let newStr = "";
+    for (let i = 0; i< str.length; i++){
+        str[i] === "a" ? newStr += "A" : newStr += str[i]
+    }
+    return newStr
+}
 
 // Uncomment out the next line to test your solution
-// runQ3Tests()
+runQ3Tests()
 
 // Question Four:
 // Write a function called largest that returns the largest value in an array
+const largest = arr =>{
+  let greatest = arr[0];
+  for (let i = 0; i < arr.length; i++){
```

Comment:
if you set the greatest to arr[0] then you have already accounted for the first element, you can start iterating with the second element, so you can set i to 1, instead of 0. Otherwise your very first iteration is comparing the first element to itself.
On the other hand if you set greatest to -Infinity (which sometimes is recommended to avoid some edge cases) you'd want to start with arr[0]